flink-table/flink-table-api-java-bridge/src/main/java/org/apache/flink/table/api/bridge/java/StreamStatementSet.java

```diff
@@ -38,18 +40,26 @@
 @PublicEvolving
 public interface StreamStatementSet extends StatementSet {
 
+    /** {@inheritDoc} */
```

Comment:
This should actually not be necessary. I remember that I also did some research. Did you verify this with IntelliJ?